### ISSUE TITLE
Pass optional name to uploadFileToDisk

### ISSUE THREAD

**Issue by mserralta** [Laravel-Backpack/CRUD#1874]
I think passing an optional extra parameter to set the name of the uploaded file could be nice. If it's ommited md5 function will determine the name as always but if it is passed it will use it.

https://github.com/Laravel-Backpack/CRUD/blob/2fcf8b6e78353118e16899e4e70e46a3ef766f2b/src/CrudTrait.php#L170

**Comment by tabacitu** [comment]
I agree. I think this would be a cool improvement - I believe a lot of people would appreciate more wiggle room in that area. I've marked it as a feature for v4.

Thanks!
